src/Snowcap/Emarsys/Client.php

```diff
@@ -264,6 +264,24 @@ public function updateContact(array $data)
     }
 
     /**
+     * Updates one or more contacts/recipients, identified by an external ID. If the contact does not exist in the
+     * database, it is created.
```

Comment:
Code comments are generally evil as they start lying with a pass of time. If you think more description is required maybe a method name is not good enough.